aleph/templates/email/api_key_expired.html

```diff
@@ -0,0 +1,15 @@
+{% extends "email/layout.html" %}
+
+{% block content -%}
+  <p>
+    {% trans expires_at=(expires_at | datetimeformat) -%}
+    Your API key has expired on {{expires_at}} UTC.
+    {%- endtrans %}
+  </p>
+
+  <p>
+    {% trans settings_url=settings_url -%}
+    If you do not use the Aleph API you can ignore this email. If you use the Aleph API you will need to <a href="{{settings_url}}">regenerate your API key</a> and update any applications or scripts that use your API key.
```

Comment:
Yes, the API can still be used without authentication in order to access public data. Do you think updating the wording as follows would clarify that sufficiently?

> If you do not use the Aleph API you can ignore this email. If you use the Aleph API to access data that isn’t public you will need to regenerate your API key and update any applications or scripts that use your API key.

Reply:
If you do not use the Aleph API, or only use the API to access public information you can ignore this email. If you use the Aleph API to access information that is not publicly accessible then you'll need to regenerate your API key in order to keep accessing the protected information.

Reply:
Just as a note for my future self: I double checked one detail about the Aleph API. When sending a request with an invalid API key, we simply ignore the API key and treat the request as unauthenticated.

This isn’t super intuitive and looking at the code it doesn’t seem like it was implemented like this intentionally. Imo it would be better to respond with an authentication error if an invalid API key is supplied, even if authentication isn’t required for a request.

But I’ve kept this behavior for backwards compatibility, and it’s a lucky coincidence that it helps with the messaging in this case.